<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QSerialFrame</class>
 <widget class="QFrame" name="QSerialFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>209</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>492</width>
    <height>209</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>492</width>
    <height>209</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>串口模拟程序</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>231</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>入口</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>22</y>
      <width>210</width>
      <height>117</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>CAN地址</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spEnterCan">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QCheckBox" name="chkEnterSense0">
         <property name="text">
          <string>地感命令</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn0">
         <property name="statusTip">
          <string>0</string>
         </property>
         <property name="text">
          <string>车入</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QCheckBox" name="chkEnterCard">
         <property name="text">
          <string>卡号</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spEnterCard">
         <property name="maximum">
          <number>16777215</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn1">
         <property name="statusTip">
          <string>1</string>
         </property>
         <property name="text">
          <string>发送</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QCheckBox" name="chkEnterSense1">
         <property name="text">
          <string>地感命令</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn2">
         <property name="statusTip">
          <string>2</string>
         </property>
         <property name="text">
          <string>车出</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>231</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>出口</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>22</y>
      <width>210</width>
      <height>117</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>CAN地址</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spLeaveCan">
         <property name="minimum">
          <number>2</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>2</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QCheckBox" name="chkLeaveSense0">
         <property name="text">
          <string>地感命令</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn3">
         <property name="statusTip">
          <string>3</string>
         </property>
         <property name="text">
          <string>车入</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QCheckBox" name="chkLeaveCard">
         <property name="text">
          <string>卡号</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spLeaveCard">
         <property name="maximum">
          <number>16777215</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn4">
         <property name="statusTip">
          <string>4</string>
         </property>
         <property name="text">
          <string>发送</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QCheckBox" name="chkLeaveSense1">
         <property name="text">
          <string>地感命令</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn5">
         <property name="statusTip">
          <string>5</string>
         </property>
         <property name="text">
          <string>车出</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>471</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>定时发送间隔</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spTimer">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
      <property name="value">
       <number>2000</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>ms</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnTimer">
      <property name="text">
       <string>启用</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
